src/TestSessionEnvironment.php

```diff
@@ -613,4 +613,15 @@ public function waitForPendingRequests($await = 700, $timeout = 10000)
 
         return true;
     }
+
+    private function connectToDB(array $databaseConfig): void
+    {
+        if (method_exists(DB::class, 'hasConfig') && DB::hasConfig('primary')) {
+            // CMS 5.4+ - ensure we connect the primary connection and not a replica
+            DB::connect($databaseConfig, 'primary');
```

Comment:
```suggestion
            DB::connect($databaseConfig, DB::CONN_PRIMARY);
```

Reply:
Also why are we checking `DB::hasConfig('primary')`? Don't we just care if the method exists (and therefore we're on 5.4+)?

Reply:
Have removed the hasConfig() check, now just checking to see if the method exists